network-store-model/src/main/java/com/powsybl/network/store/model/AbstractTapChangerStepAttributes.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public abstract class AbstractTapChangerStepAttributes {
```

Comment:
Would it have been simpler to only have one class TapChangerStepAttributes ? and for a ratio tap changer usage the alpha is not used.

Reply:
I didn't want to edit too much the current model but it could simplify a lot of code yes.
Will think about this refactoring in another PR.